play/symphony-conductor/js/app.js

```diff
@@ -122,6 +140,15 @@ const INSTRUMENTS = [
         code: `play("Gong");`,
         link: "https://en.wikipedia.org/wiki/Gong"
         
+    },
+    {
+        name: 'Hand-Clap',
+        description: "Hand-Clap: Considered a percussion instrument",
+        img: 'img/handclap.png',
+        sound: 'sounds/handclap.wav',
+        code: `play("Hand-Clap");`,
```

Comment:
Could you remove the capitalization in the code to match the rest of the instruments?